ecmd-core/capi/ecmdUtils.C

```diff
@@ -764,21 +764,20 @@ uint32_t ecmdDisplayScomData(const ecmdChipTarget & i_target, const ecmdScomData
   if ((std::string)i_format == "-vs1") {
     verboseBitsSetFlag = true;
   }
-  rc = ecmdQueryFileLocation(i_target, ECMD_FILE_SCOMDATA, l_filePairs, l_version);
+  rc = ecmdQueryFileLocation(i_target, ECMD_FILE_SCOMDATA, l_fileLocs, l_version);
   if (rc) {
     printed = "ecmdDisplayScomData - Error occured locating scomdef file \nSkipping -v parsing\n";
     ecmdOutputWarning(printed.c_str());
     return rc;
   }
 
   std::ifstream scomdefFile;
-  std::list<std::pair<std::string, std::string> >::iterator l_filePair = l_filePairs.begin();
-  while (l_filePair != l_filePairs.end())
+  for (auto l_fileLoc : l_fileLocs)
```

Comment:
This is going to cause problems with some of the old 32 bit compilers for architectures we support.